crypto/encode.go

```diff
@@ -237,13 +237,13 @@ func deCompress(yTilde int, xValue []byte, curve *elliptic.CurveParams) (*PubKey
 }
 
 func DecodePoint(encodeData []byte) (*PubKey, error) {
+	if nil == encodeData {
```

Comment:
len(encodeData) == 0 should be better since it handles both nil and non-nil but empty case

Reply:
thanks. Done.